linera-views/src/scylla_db.rs

```diff
@@ -148,6 +152,48 @@ impl ScyllaDbClient {
         })
     }
 
+    async fn read_multi_values_internal(
+        &self,
+        keys: Vec<Vec<u8>>,
+    ) -> Result<Vec<Option<Vec<u8>>>, ScyllaDbContextError> {
+        let num_keys = keys.len();
+        if num_keys == 0 {
+            return Ok(Vec::new());
+        }
+        let session = &self.session;
+        let mut group_query = "?".to_string();
+        group_query.push_str(&",?".repeat(num_keys - 1));
+        let query = format!(
+            "SELECT k,v FROM kv.{} WHERE dummy = 0 AND k IN ({}) ALLOW FILTERING",
+            self.namespace, group_query
+        );
+        let read_multi_value = Query::new(query);
+        let mut rows = session.query_iter(read_multi_value, &keys).await?;
+        let mut values = Vec::with_capacity(num_keys);
+        values.resize(num_keys, None);
+        let mut map = HashMap::<Vec<u8>, Vec<usize>>::new();
+        for (i_key, key) in keys.into_iter().enumerate() {
+            match map.entry(key) {
+                Entry::Occupied(entry) => {
+                    let entry = entry.into_mut();
+                    entry.push(i_key);
+                }
+                Entry::Vacant(entry) => {
+                    entry.insert(vec![i_key]);
+                }
+            }
+        }
+        while let Some(row) = rows.next().await {
+            let value = row?.into_typed::<(Vec<u8>, Vec<u8>)>()?;
+            let key = value.0;
+            for i_key in map.get(&key).unwrap().clone() {
+                let value = Some(value.1.clone());
+                *values.get_mut(i_key).unwrap() = value;
```

Comment:
In production code:
`.unwrap()` -> `.expect("some reason why this should work)`

Reply:
Ok.